src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer.java

```diff
@@ -40,6 +56,39 @@ public void taskCompletedWithProblems(Executor executor, Queue.Task task, long d
         LOGGER.log(Level.FINE, " Computer " + this + " taskCompletedWithProblems");
     }
 
+    @Exported
+    public List<Container> getContainers() throws UnrecoverableKeyException, CertificateEncodingException, NoSuchAlgorithmException, KeyStoreException, IOException {
+        KubernetesSlave slave = getNode();
```

Comment:
permission checks are missing, not sure if READ permission should be enough to access such data

Reply:
Yes, the only administrator can get this data. Thanks.